UnicodeJsps/src/main/webapp/footer.jsp

```diff
@@ -1,11 +1,16 @@
-<%@ page import="org.unicode.jsp.*" %>
-<hr>
-<p style="font-size:80%"><b><a name="fonts">Fonts and Display.</a></b> If you don't have a good set of Unicode fonts (and modern browser), 
+<%@ page import="org.unicode.jsp.*,java.time.Year" %>
+<footer>
+    <p><b><a name="fonts">Fonts and Display.</a></b> If you don't have a good set of Unicode fonts (and modern browser),
 you may not be able to read some of the characters.
 Some suggested fonts that you can add for coverage are:
 <a href="https://www.google.com/get/noto/" target="_blank">Noto Fonts site</a>,
 <a href="http://greekfonts.teilar.gr/" target="_blank">Unicode Fonts for Ancient Scripts</a>,
 <a href="http://www.alanwood.net/unicode/fonts.html" target="_blank">Large, multi-script Unicode fonts</a>.
 See also: <a href="http://www.unicode.org/help/display_problems.html" target="_blank">Unicode Display Problems</a>.</p>
-<p style="font-size:80%"><%= UnicodeJsp.getVersions() %></p>
-<hr>
\ No newline at end of file
+<p><%= UnicodeJsp.getVersions() %> &#x2022; <a href="https://github.com/unicode-org/unicodetools">source</a></p>
+    <p>
+<!-- yes, UTF-8 issue here, so using &copy; -->
+&copy; 2001-<%= Year.now() %> Unicode, Inc. Unicode and the Unicode Logo are registered trademarks of Unicode, Inc. in
+    the U.S. and other countries. See <a href="https://www.unicode.org/copyright.html">Terms of Use</a>.
```

Comment:
Should this say "For terms of use and license" as usual?
And point to https://www.unicode.org/terms_of_use.html like we do elsewhere?

Reply:
I'm not sure about 'elsewhere' or 'as usual' but I'm using the updated approved language, matching several other projects that @annebright has approved:
<img width="575" alt="image" src="https://github.com/user-attachments/assets/87bf4db8-c0c9-497b-9795-7a3012dc292a" />

Reply:
terms_of_use.html is a redirect, I don't think it's used in updated notices.

Reply:
It's just a bit different from when we worked with Anne to revamp the copyright headers in the data files. 🤷

Reply:
It is different from the data files… but, this isn't a data file.